Kill useless libraries:
 - ${CMAKE_DL_LIBS} and pthread are already added
 - gfortran is not used

--- src/CMakeLists.txt	2013-11-18 09:24:32.537295000 +0100
+++ ../robotpkg/motion/sot-core/work.gupta/sot-core-2.9.0/src/CMakeLists.txt	2013-11-18 16:20:37.785839000 +0100
@@ -233,9 +233,6 @@
     TARGET_LINK_LIBRARIES(${LIBRARY_NAME} ${CMAKE_DL_LIBS})
   ENDIF(UNIX)
 
-  IF(UNIX AND NOT APPLE)
-    TARGET_LINK_LIBRARIES(${LIBRARY_NAME} dl pthread)
-  ENDIF(UNIX AND NOT APPLE)
 
   PKG_CONFIG_USE_DEPENDENCY(${LIBRARY_NAME} jrl-mal)
   PKG_CONFIG_USE_DEPENDENCY(${LIBRARY_NAME} dynamic-graph)
